palace/models/surfaceconductivityoperator.hpp

```diff
@@ -1,8 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef PALACE_SURF_CONDUCTIVITY_OPERATOR_HPP
-#define PALACE_SURF_CONDUCTIVITY_OPERATOR_HPP
+#ifndef PALACE_MODELS_SURF_CONDUCTIVITY_OPERATOR_HPP
+#define PALACE_MODELS_SURF_CONDUCTIVITY_OPERATOR_HPP
```

Comment:
Surf should be Surface, the shortening is inconsistent with other head blockers. This holds for all the SURF_ in the other header blockers too.

Reply:
Addressed in [ec816d9](https://github.com/awslabs/palace/pull/46/commits/ec816d9b22aee3d31f7bae3f54677f06374eff06).